web-ui/src/pages/ReviewsPage.jsx

```diff
@@ -1,43 +1,46 @@
-import React, {useCallback, useEffect, useState} from "react";
-import { styled } from '@mui/material/styles';
```

Comment:
Nearly all the changes in this file are from Prettier formatting. I'll add comments below that describe the real changes.